skeleton/cypress/aurelia_project/tasks/cypress.json

```diff
@@ -6,6 +6,23 @@
       "name": "run",
       "description": "Perfom a testrun and reports the results.",
       "type": "boolean"
+    },
+    // @if feat.webpack
+    {
+      "name": "host",
+      "description": "Set host address of the dev server, the accessible URL",
+      "type": "string"
+    },
+    // @endif
+    {
+      "name": "port",
+      "description": "Set port number of the dev server",
+      "type": "string"
+    },
+    {
+      "name": "start",
+      "description": "Runs the application before starting to perform a testrun",
+      "type": "boolean"
```

Comment:
Did you forget push up your changes on cypress.ext?

Reply:
That supposed to be in another PR, however, I could merge the [branch](https://github.com/shahabganji/cli/tree/feature/e2e_cypress) into master and send all in this PR, but the PR, should be renamed then for the sake of the history of the repository. Check it out and let me know of your opinion.